docs/csharp/programming-guide/concepts/linq/quantifier-operations.md

```diff
@@ -19,6 +19,23 @@ Quantifier operations return a <xref:System.Boolean> value that indicates whethe
 |All|Determines whether all the elements in a sequence satisfy a condition.|Not applicable.|<xref:System.Linq.Enumerable.All%2A?displayProperty=nameWithType><br /><br /> <xref:System.Linq.Queryable.All%2A?displayProperty=nameWithType>|  
 |Any|Determines whether any elements in a sequence satisfy a condition.|Not applicable.|<xref:System.Linq.Enumerable.Any%2A?displayProperty=nameWithType><br /><br /> <xref:System.Linq.Queryable.Any%2A?displayProperty=nameWithType>|  
 |Contains|Determines whether a sequence contains a specified element.|Not applicable.|<xref:System.Linq.Enumerable.Contains%2A?displayProperty=nameWithType><br /><br /> <xref:System.Linq.Queryable.Contains%2A?displayProperty=nameWithType>|  
+
+## Query Expression Syntax Examples  
+  
+### All  
+The following example uses the `All` to check that all strings are of a specific length.
+  
+[!code-csharp-interactive[All](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQQuantifier/CS/Quantifier.cs#All)]  
```

Comment:
The samples won't run correctly in the current interactive experience. (There will be new features coming in the next month or two that will change that). In the meantime, let's change the include to not run the samples:

```suggestion
[!code-csharp[All](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQQuantifier/CS/Quantifier.cs#All)]  
```